### PR TITLE
A17: Sidebar

### PR DESCRIPTION
**Pull request by Angelbt12**
sidebar finished with html and css

### CODE REVIEW

File: src/ab/html/A2-Markup-a-readme/ab/A17-Sidebar/sidebar1.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/sidebar1.css" />
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/ab/A17-Sidebar/sidebar1.html:6]: The page is not operating as expected, please check and correct.

**Comment by luisalejandrosalazar** [comment]
The page is not operating as expected, please check and correct.

image

File: src/ab/html/A2-Markup-a-readme/ab/A17-Sidebar/sidebar1.html (hunk 1 above)
Comment (Angelbt12) [reply src/ab/html/A2-Markup-a-readme/ab/A17-Sidebar/sidebar1.html:6]: fix made was to change the css path

**Review by luisalejandrosalazar — APPROVED** [review]
Approved

**Pull request merged**